doc/development/contributing.rst

```diff
@@ -215,7 +215,7 @@ Once you have git installed and configured, and before creating your local copy
 of the codebase, go to the `MNE-Python GitHub`_ page and create a `fork`_ into
 your GitHub user account.
 
-.. image:: https://docs.github.com/assets/cb-28613/images/help/repository/fork_button.png
+.. image:: https://docs.github.com/assets/cb-79331/mw-1440/images/help/repository/fork-button.webp
 
```

Comment:
the new image is really different from the prior one. I think we should just delete the image, it doesn't add a lot.

```suggestion
```